app/src/main/java/com/example/groupproject/MainActivity.kt

```diff
@@ -6,6 +6,6 @@ import android.os.Bundle
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        setContentView(R.layout.clock_timer)
```

Comment:
don't set the context to clock_timer keep it as activity_main, ideally, we need to merge activity_main and clock_timer.